tinynn/converter/operators/torch/aten.py

```diff
@@ -2388,3 +2388,20 @@ def parse(self, node, attrs, args, graph_converter):
 
         self.run(node)
         ATenToOperator.parse_common(self, node, attrs, args, graph_converter)
+
+
+class ATenTopkOperator(ATenTopkSchema):
+    def parse(self, node, attrs, args, graph_converter):
+        super().parse(node, attrs, args, graph_converter)
+
+        self.run(node)
+        input_tensor, k, dim, largest, sorted = self.input_tensors
+        log.warning(
+            "ATenTopkOperator only return the top k largest element along '            'each last dimensional slice of"
+            " input and the indices of values within '            'the last dimension of the input tensor"
+        )
+        input_tensor, k = [self.find_or_create_input(i, graph_converter) for i in range(2)]
```

Comment:
Why not just assign the values to a single variable `inputs`? Also, I wonder whether this works since `k` is of type `int`.